.github/workflows/frontend.yml

```diff
@@ -14,6 +14,8 @@ jobs:
 
     env:
       CI: true
+      BACKSTAGE_CACHE_DIR: <repoRoot>/.backstage-build-cache
```

Comment:
don't want in node modules because it conflicts with their (separate) caching concept i guess?

Reply:
Yeah this cache has a separate lifespan than all of node_modules